src/tests-tokens/tokens-metadata-tests.ts

```diff
@@ -98,16 +97,17 @@ describe('api tests', () => {
       postConditionMode: PostConditionMode.Allow,
       sponsored: false,
       anchorMode: AnchorMode.Any,
+      fee: 100000,
     });
 
     const contractId = senderAddress + '.' + contractName;
 
-    const feeRateReq = await fetch(stacksNetwork.getTransferFeeEstimateApiUrl());
-    const feeRateResult = await feeRateReq.text();
-    const txBytes = new BN(contractDeployTx.serialize().byteLength);
-    const feeRate = new BN(feeRateResult);
-    const fee = feeRate.mul(txBytes);
-    contractDeployTx.setFee(fee);
+    // const feeRateReq = await fetch(stacksNetwork.getTransferFeeEstimateApiUrl());
+    // const feeRateResult = await feeRateReq.text();
+    // const txBytes = BigInt(contractDeployTx.serialize().byteLength);
+    // const feeRate = BigInt(feeRateResult);
+    // const fee = feeRate * txBytes;
+    // contractDeployTx.setFee(fee);
     const { txId } = await sendCoreTx(contractDeployTx.serialize());
```

Comment:
The commented out code above never seemed to work as intended (including in v4). It attempted to set the fee _after_ the tx was already signed, which would result in the stacks-node rejecting the transaction because of signature mismatch.

Which means either A) the `.setFee(fee)` function wasn't actually changing the tx data, or B) it happened to be setting the fee to the same value already calculated by stacks.js during the `await makeContractDeploy(...)` call.